assets/javascripts/discourse/components/activity-pub-visibility-dropdown.js

```diff
@@ -0,0 +1,28 @@
+import DropdownSelectBoxComponent from "select-kit/components/dropdown-select-box";
+import I18n from "I18n";
+import { computed } from "@ember/object";
+
+export default DropdownSelectBoxComponent.extend({
```

Comment:
This looks odd locally for me: 

<img width="735" alt="image" src="https://github.com/discourse/discourse-activity-pub/assets/368961/d76e4f02-fac5-4411-a82c-b7f43522a65f">

I believe it's because `DropdownSelectBoxComponent` is used in slightly different circumstances. It may be more fitting to use: 

```
import ComboBoxComponent from "select-kit/components/combo-box";
```

that will result in a regular dropdown, like the Subcategory List Style etc. (higher up in the same category edit screen).

Reply:
Sure. I used the DropdownSelectBoxComponent as it was more appropriate for the composer, and the same component was used across both. But, yes, when just in the category settings a ComboBox makes more sense.

Reply:
Done https://github.com/discourse/discourse-activity-pub/pull/19/commits/d6565a4cac9d8f17a51c083601e4256728dd08df